scripts/restore.sh

```diff
@@ -344,6 +344,9 @@ restore_active_and_alternate_sessions() {
 main() {
 	if supported_tmux_version_ok && check_saved_session_exists; then
 		start_spinner "Restoring..." "Tmux restore complete!"
+		if [ -n "$(restore_hook)" ]; then
+			eval "$(restore_hook)"
+		fi
```

Comment:
Tricksy one - I would like a hook that is run for each pane. Once you add one hook people quickly want a hook everywhere, eh?